docs-md/config/eventer.md

```diff
@@ -72,18 +72,26 @@ The keys and values supported are:
 
    This establishes a new named jobq and sets parameters around concurrency
    and memory safety.  The format of the value is:
-   `concurrency[,min[,max[,safety[,backlog]]]]`.  Concurrency, min, max, and backlog are all
+   `concurrency[,min[,max[,safety[,backlog[,lifo]]]]]`.  Concurrency, min, max, and backlog are all
    unsigned integers. Concurrency must be greater than zero.  If minimum is omitted or
    zero, no minimum is set.  If max is omitted, it is set to min.  If max is
    zero, there is no maximum.  Safety can be one of `none` , `cs` , or
    `gc` (default).  For more information om memory settings see [eventer_jobq.h](https://github.com/circonus-labs/libmtev/tree/master/src/eventer/eventer_jobq.h) and [mtev_memory.h](https://github.com/circonus-labs/libmtev/tree/master/src/utils/mtev_memory.h). Backlog sets
-   the advisory queue length backlog limit for the queue.
+   the advisory queue length backlog limit for the queue. The `lifo` setting instructs the
+   jobq to process event last-in-first-out.  The values for this field is either `lifo` or 
+   `fifo`.
 
    > Note that this merely creates the jobq. One must find and use it
    > programmatically within the software.  It is designed to have a code-free
    > way of allowing operators to adjust parameters around jobqs used within
    > an application.
 
+ * ##### default_jobq_ordering
+
+   Specified if jobqs should, by default, use LIFO or FIFO processing order.  The default
```

Comment:
", by default," seems confusing here.  The "default" (i.e., no value given) is FIFO, otherwise it respects the setting, so I think it's cleaner without that clause.